Sources/Exhibition/DebugView.swift

```diff
@@ -23,16 +25,38 @@ struct DebugView: View {
                         Text("Left to Right").tag(LayoutDirection.leftToRight)
                         Text("Right to Left").tag(LayoutDirection.rightToLeft)
                     }
+                    
+                    Button("Reset") {
+                        preferredColorScheme = .light
```

Comment:
@mjarvis does it will resets to `light` even if the user device system default color scheme is `dark` right?

Reply:
Yes -- At the moment, Exhibition uses explicit accessibility settings, rather than inheriting the device settings.